src/store/pageview.ts

```diff
@@ -29,6 +61,7 @@ class ViewStore {
       ...metadata,
     };
     this.pageViews.push(pageView);
+    this._trim();
```

Comment:
We do not need to trim 3x per page view :)

Reply:
Explained above.